### ISSUE TITLE
Specify table name in obfuscate calls

### ISSUE THREAD

**Issue by nettofarah** [IFTTT/polo#29]
As of today, we just blindly try to match a global list of fields we want to be obfuscated against all the attributes of every row Polo traverses:
https://github.com/IFTTT/polo/blob/b79e48e897c9a552cce64f0b9fd6f36e569035c0/lib/polo/translator.rb#L38

To do so, we pass in a list of fields to `Polo.configure`
```ruby
Polo.configure do
  obfuscate :email
end
```

While this has been working well, it could be the case that someone wants to only obfuscate certain fields in certain tables. 

What I'm suggesting is that we change the `obfuscate` method to also accept something like this:

```ruby
Polo.configure do
  obfuscate "users.email" 
end
``` 
So in this case we're only obfuscating `email` fields in the `users` table, and not obfuscating emails on some other tables such as `invites` or some other table where emails are not sensitive.

**Comment by nettofarah** [comment]
What do you think, @danielvlopes?

**Comment by craigmcnamara** [comment]
How about
```
Polo.configure do
  email_scrubber = ->(e) { # secret stuff }
  pure_nonsense = -> (v) { # make nonsense }
  obfuscate users: { email: email_scrubber },
                   credit_cards: { number: pure_nonsense, :cvc }
end
```

**Comment by nettofarah** [comment]
This looks even better. Though I think this would break the existing api.

**Comment by craigmcnamara** [comment]
I needed this feature so I made this https://github.com/IFTTT/polo/pull/30

**Comment by nettofarah** [comment]
Solved by #30
